src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -263,6 +259,9 @@ export default {
 				this.passwordProtect = false
 			}
 		},
+		showModal(newVal) {
```

Comment:
We can't change `props` directly, like we doing with `data`
Instead, we need to emit a new value, 'false' in our case, and capture it in `LeftSidebar` with @close="toggle(false)", like it works now